### DISCUSSION TITLE
It doesn't work under Debian Linux

### DISCUSSION THREAD

**Discussion by gerroon** [LazyVim/LazyVim#5216] [Q&A]
Hi

I installed it under WSL2 Debian, I have nvim and lazyvim (installed using the install page commands). For some reason when I start nvim, I see no lazyvim or install activaty.

`NVIM v0.4.4`

```
~/.config/nvim$ ls
LICENSE  README.md  init.lua  lua  stylua.toml

```

```
:version
NVIM v0.4.4
Build type: Release
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/cc -g -O2 -fdebug-prefix-map=/build/neovim-eK2ojg/neovim-0.4.4=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=1 -DDISABLE_LOG -Wdate-time -D_FORTIFY_SOURCE=1 -O2 -DNDEBUG -DMIN_LOG_LEVEL=3 -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -I/build/neovim-eK2ojg/neovim-0.4.4/build/config -I/build/neovim-eK2ojg/neovim-0.4.4/src -I/usr/include -I/usr/include/lua5.1 -I/build/neovim-eK2ojg/neovim-0.4.4/build/src/nvim/auto -I/build/neovim-eK2ojg/neovim-0.4.4/build/include
Compiled by team+vim@tracker.debian.org

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"


```

**Comment by dpetka2001** [comment]
Read the minimum requirements.

**Comment by netskink** [comment]
Hmm.  I have the same problem.  I can use it on MacOS, but not on a Debian box.

I read the page here for the minimum requirements.  I assume this is the page you specifically want us to read.

https://www.lazyvim.org

With that said, here is what I installed:

* lua5.4
* git 2.30.2
* apt search lazygit did not remove anything.
* ditto for nerdfont.  I've installed this on macOS previously when I saw the ? chars.  However, I don't see this font in search on debian stable.
* curl was already installed
* nvim 0.4.4  (yikes)
* lua-vim
* python3-pynvim

Upon doing this, I noticed the nvim release is too old.  It does mention JIT enables.  Likewise the lua-vim.   However, this might be why it does work - the nvim version is too  old.

**Reply by folke** [reply]
nvim 0.4.4 doesn't even have the lua API yet (that was since 0.5.0),
so super ancient version from the stoneage.

You need at least 0.9 for LazyVim.

**Reply by netskink** [reply]
Yes, as I was sying "the nvim version is too old".  Oh, the joy's of Debian Stable.

**Reply by Jacobus-afk** [reply]
you can use these install [instructions](https://github.com/nvim-lua/kickstart.nvim#linux-install) to get the latest(nightly) version of neovim, if you haven't come right yet..

just choose the option for debian

**Reply by netskink** [reply]
cool will try that.  Thanks!
